js/components/circe/components/CohortConceptSetBrowser.js

```diff
@@ -81,6 +77,9 @@ define(['knockout', 'text!./CohortConceptSetBrowserTemplate.html', 'services/Voc
 
 			VocabularyProvider.getConceptSetList(url)
 				.done(function (results) {
+					results.forEach(cs => {
```

Comment:
These 3 lines are repeated at least 7 times. Wouldn't it be better to move them into Utils?